ras_common_utils/ras_config/ras_config.py

```diff
@@ -62,54 +70,82 @@ def items(self):
         return self.service.items()
 
     def dependency(self, k):
+
         try:
             return DependencyProxy(self._dependencies[k], k)
         except KeyError:
             raise RasDependencyError("Dependency with name '{}' not found.".format(k))
 
     def dependencies(self):
+
         return {k: DependencyProxy(self._dependencies[k], k) for k in self._dependencies.keys()}.items()
 
     def features(self):
         return self._features
 
+    def is_cloud_foundry(self):
+        return self._is_cloud_foundry
+
 
 class CloudFoundryServices:
     def __init__(self, service_data):
-        self._lookup = {v['name']: v['credentials']
+
+        # Extract the Cloud Foundry settings that describe the DB connection which looks like this:
+        # {'ras-ps-db': {'host': 'my_long_host_name.rds.amazonaws.com', 'password': 'my_secret_password',
+        # 'username': 'my_long_hostname',
+        # 'uri': 'postgres://my_username:my_password@my_hostname.rds.amazonaws.com:5432/db3d0u371x4tidjnu', 'db_name': 'my_demo_database'}}
+
+        temp_cloud_foundry_settings = {v['name']: v['credentials']
                         for service_config in service_data.values()
                         for v in service_config}
 
+        self._lookup = {}
+        # Grab the first key value and populate the lookup 'acitve_db_connection' private variable
+        for key in temp_cloud_foundry_settings:
+            self._lookup['active_db_connection'] = temp_cloud_foundry_settings[key]
```

Comment:
Do we need this? The lookup dict allows us to couple against a specific CF service via the name we give it in the deployment. E.g. 'ras-party-db'. If we pick out the first key from the lookup table to use as the 'active db connection', this can be wrong in certain circumstances, such as when we have more than one CF service.